src/main/resources/assets/bloodmagic/lang/en_us.json

```diff
@@ -0,0 +1,504 @@
+{
+  "guide.bloodmagic.name": "§4§lSanguine Scientiem",
+  "guide.bloodmagic.subtitle": "§9§oAlchemical Wizardry",
+  "guide.bloodmagic.landing": "Welcome to $(6)Blood Magic$()! $(br2)$(l:bloodmagic:utility/nyi)A lot of stuff$() isn't yet implemented, so please excuse our dust. $(br2)Click $(l:bloodmagic:utility/getting_started)HERE$() to get started. If you find any bugs, please report them on our $(l:https://github.com/WayofTime/BloodMagic/issues)Github$().)",
```

Comment:
The en_us.json file is a generated file.  It is located at `src/generated/resources/assets/bloodmagic/lang/en_us.json`.

The Data Generator that makes it is located at `src/main/java/wayoftime/bloodmagic/common/data/GeneratorLanguage.java`.

This non-generated en_us file should be deleted, these lines (I think the only ones added) should be moved to the Data Generator ([I recommend this section](https://github.com/WayofTime/BloodMagic/blob/f744078f63db5f5b20a08a92a9551db1302b627f/src/main/java/wayoftime/bloodmagic/common/data/GeneratorLanguage.java#L376), which may have been added after you started this translation), and running the data generator to add them to the generated file.

Let me know if you want me to make and PR those changes to your fork.

I don't have any issues with the non-generated ru_ru file.  It should work as-is (other than maybe tweaking the resource names to match the more recently added entries using `patchouli.bloodmagic.<relevant name>`).